### DISCUSSION TITLE
Running Pluto on a server

### DISCUSSION THREAD

**Discussion by NiallPalfreyman** [JuliaPluto/PlutoTest.jl#24] [Q&A] [answered]
Hi. I'm wanting to make available small simulations, quizzes, explanations available as part of a marketing exercise for a university degree programme. I want to set these programs up in Pluto notebooks, but I don't want clients to have to install Julia/Pluto on their own machine. Rather, I might for instance want prospective students to be able to interact with a planetary motion or disease propagation simulation that runs on our server.

So what I'm looking for is a way to set up Pluto notebooks to run on our server, but make available interactive capability to people using nothing more than a local web browser. Is this easily possible?

**Comment by fonsp** [comment, marked as answer]
Hi Niall! It sounds like you are looking for https://github.com/JuliaPluto/PlutoSliderServer.jl ! You can see an example here: https://margo.plutojl.org/ . This can be a bit difficult to set up, I would only recommend it if you have set up some other kind of server before.

If not, you can use binder to put interactive material online. The easiest way is to export your notebook to HTML, and to let students click the "Edit & run this notebook" button in the top right. Using small packages will really help here. You can learn more about this in my juliacon talk: https://www.youtube.com/watch?v=HiI4jgDyDhY&t=473s

Let me know if this worked!

All the best
-fons
